2020-04-01 09:17:52,969 [qtp917819120-406] session-id=[472] user-id=[-1] robot-name=[ev3lejosv1] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] init gallery list view
2020-04-01 09:18:52,969 [qtp917819120-406] session-id=[472] user-id=[-1] robot-name=[ev3lejosv1] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] init program delete
2020-04-01 09:20:59,648 [qtp917819120-1383] session-id=[4591] user-id=[-1] robot-name=[calliope2017NoBlue] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] run NEPOprogon brick
2020-04-15 09:27:34,384 [qtp917819120-621] session-id=[2072] user-id=[-1] robot-name=[ev3dev] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[TIME]] 89 msec: tabConfiguration clicked
2020-04-01 09:30:32,634 [qtp917819120-517] session-id=[] user-id=[] robot-name=[] ERROR d.f.i.r.j.p.JAXBProviderFactory - marshaller not valid for type com.sun.research.ws.wadl.Application
2020-04-01 09:31:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login
2020-04-01 09:32:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login
2020-04-01 09:33:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login
2020-04-01 09:34:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.p.DbcKeyExceptionMapper - init exception was caught at system border: invalid init token
2020-04-01 09:35:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.p.DbcKeyExceptionMapper - init exception was caught at system border: invalid init token
2020-04-01 09:36:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] INFO d.f.i.r.p.AbstractProcessor - Processor failed: USER_GET_ONE_ERROR_ID_OR_PASSWORD_WRONG
2020-04-01 09:37:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] INFO d.f.i.r.p.AbstractProcessor - Processor failed: PROGRAM_GET_ONE_ERROR_NOT_FOUND
2020-04-01 09:38:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.w.CalliopeCompilerWorker - compile calliope2017NoBlue program NEPOprog failed with (COMPILERWORKFLOW_ERROR_PROGRAM_COMPILE_FAILED, [1/3] Building the NEPOprog.
2020-04-01 09:39:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.w.c.ArduinoCompilerWorker - compile nano program FahrenGrundlagenundUltraschallsensor failed with (COMPILERWORKFLOW_ERROR_PROGRAM_COMPILE_FAILED, /tmp/openrobertaTmp/CNPEY2U3/FahrenGrundlagenundUltraschallsensor/source/FahrenGrundlagenundUltraschallsensor.ino: In function 'void setSpeedLeft(double)':
2020-04-01 09:40:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.c.ClientConfiguration - Exception. Error ticket: E-55
2020-04-01 09:41:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.persistence.dao.ProgramDao - update was requested, timestamps don't match. Has another user updated the program in the meantime?
2020-04-01 09:42:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.r.RobotCommunicator - ROBOT_RC: /pushcmd from a robot arrived, no matching state was found - we return a server error
2020-04-01 09:43:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.persistence.dao.ProgramDao - update was requested, timestamps don't match. Has another user updated the program in the meantime?
2020-04-01 09:44:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.r.RobotDownloadProgram - upload error: file 'NEPOprog.hex' to upload to robot not found.
2020-04-01 09:45:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.w.c.SenseboxCompilerWorker - compile sensebox program NEPOprog failed with (COMPILERWORKFLOW_ERROR_PROGRAM_COMPILE_FAILED, /tmp/openrobertaTmp/f2MV0_x3t-O_/NEPOprog/source/NEPOprog.ino: In function 'void loop()':
2020-04-01 09:47:52,969 [qtp917819120-406] session-id=[472] user-id=[-1] robot-name=[ev3lejosv1] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] init gallery list view
2020-04-01 09:48:52,969 [qtp917819120-406] session-id=[472] user-id=[-1] robot-name=[ev3lejosv1] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] init program delete
2020-04-01 09:50:59,648 [qtp917819120-1383] session-id=[4591] user-id=[-1] robot-name=[calliope2017NoBlue] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[INFO]] run NEPOprogon brick
2020-04-15 09:51:34,384 [qtp917819120-621] session-id=[2072] user-id=[-1] robot-name=[ev3dev] INFO d.fhg.iais.roberta.util.ClientLogger - log entry: [[TIME]] 89 msec: tabConfiguration clicked
2020-04-01 09:52:32,634 [qtp917819120-517] session-id=[] user-id=[] robot-name=[] ERROR d.f.i.r.j.p.JAXBProviderFactory - marshaller not valid for type com.sun.research.ws.wadl.Application
2020-04-01 09:53:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login
2020-04-01 09:54:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login
2020-04-01 09:55:33,884 [qtp917819120-581] session-id=[1405] user-id=[76134] robot-name=[nano] ERROR d.f.i.r.j.r.a.controller.ClientUser - Invalid command: login